### ISSUE TITLE
KSQL should display failure state in `DESCRIBE EXTENDED`

### ISSUE THREAD

**Issue by rmoff** [confluentinc/ksql#2357]
Source data in JSON, mapped as a table: 

```
ksql> SET 'auto.offset.reset' = 'earliest';
Successfully changed local property 'auto.offset.reset' from 'earliest' to 'earliest'.
ksql>
ksql> SELECT * FROM COMPUTER_T LIMIT 3;
1547657372925 | 100 | 1 | 100 | e0:a1:d7:18:c2:72
1547657372925 | 101 | 2 | 101 | e1:a3:d7:18:c2:72
1547657372925 | 102 | 3 | 102 | e2:a4:d7:18:c2:72
Limit Reached
Query terminated
```

Create a copy, with Avro: 

```
ksql> CREATE TABLE FOO_AVRO WITH (VALUE_FORMAT='AVRO') AS SELECT * FROM COMPUTER_T;

 Message
---------------------------
 Table created and running
---------------------------
```

Where's my data? 

```
ksql> SELECT * FROM FOO_AVRO LIMIT 3;
^CQuery terminated
ksql>
```

Check `DESCRIBE EXTENDED`: 

```
ksql> DESCRIBE EXTENDED FOO_AVRO;

Name                 : FOO_AVRO
Type                 : TABLE
Key field            : EMPKEY
Key format           : STRING
Timestamp field      : Not set - using <ROWTIME>
Value format         : AVRO
Kafka topic          : FOO_AVRO (partitions: 4, replication: 1)

 Field   | Type
-------------------------------------
 ROWTIME | BIGINT           (system)
 ROWKEY  | VARCHAR(STRING)  (system)
 COMP_ID | INTEGER
 EMPKEY  | VARCHAR(STRING)
 MACADDR | VARCHAR(STRING)
-------------------------------------

Queries that write into this TABLE
-----------------------------------
CTAS_FOO_AVRO_3 : CREATE TABLE FOO_AVRO WITH (VALUE_FORMAT='AVRO') AS SELECT * FROM COMPUTER_T;

For query topology and execution plan please run: EXPLAIN <QueryId>

Local runtime statistics
------------------------


(Statistics of the local KSQL server interaction with the Kafka topic FOO_AVRO)
ksql>
```

Looks OK. Where's my data? Check the KSQL Server log

```
…
 [2019-01-16 17:18:53,977] ERROR Failed to send HTTP request to endpoint: http://localhost:8081/subjects/FOO_AVRO-value/versions (io.confluent.kafka.schemaregistry.client.rest.RestService:182)
 java.net.ConnectException: Connection refused (Connection refused)
    at java.net.PlainSocketImpl.socketConnect(Native Method)
    at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
    at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
    at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
    at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
    at java.net.Socket.connect(Socket.java:589)
…
[2019-01-16 17:18:53,978] ERROR task [0_0] Failed to flush state store KSTREAM-AGGREGATE-STATE-STORE-0000000004:  (org.apache.kafka.streams.processor.internals.ProcessorStateManager:222)
org.apache.kafka.common.errors.SerializationException: Error serializing row to topic FOO_AVRO using Converter API
Caused by: org.apache.kafka.connect.errors.DataException: FOO_AVRO
   at io.confluent.connect.avro.AvroConverter.fromConnectData(AvroConverter.java:78)
   at io.confluent.ksql.serde.connect.KsqlConnectSerializer.serialize(KsqlConnectSerializer.java:41)
   at io.confluent.ksql.serde.connect.KsqlConnectSerializer.serialize(KsqlConnectSerializer.java:24)
   at io.confluent.ksql.serde.tls.ThreadLocalSerializer.serialize(ThreadLocalSerializer.java:38)
   at io.confluent.ksql.serde.tls.ThreadLocalSerializer.serialize(ThreadLocalSerializer.java:24)
…
```

Turns out Schema Registry isn't running—but from a user point of view, there is *no* indication of that. 

`DESCRIBE EXTENDED` is the obvious place in which to surface this kind of problem. `SHOW QUERIES` would also be a good place to include a column showing `FAILED` state if appropriate.

**Comment by rmoff** [comment]
related: https://github.com/confluentinc/ksql/issues/3208

**Comment by big-andy-coates** [comment]
Hey @rmoff, with #3208 done, can you see if you get the feedback you require, please?

**Comment by rmoff** [comment]
Two comments: 

1. I'd suggest changing the wording in `DESCRIBE EXTENDED` from:

    For query topology and execution plan please run: EXPLAIN <QueryId>

to 

    For query status, topology, and execution plan please run: EXPLAIN <QueryId>

So that users know to go and look there for status. 

2. Ideally, we would make user's lives one step easier by surfacing the status into `DESCRIBE EXTENDED` too - the majority will want to know the query status, rather than the nitty gritty of explain plan. If they have to run EXPLAIN for each query it starts to become tedious to track down a problem. An example of how this could look would be: 

```
Queries that write into this TABLE
-----------------------------------
CTAS_FOO_AVRO_3 (REBALANCING): CREATE TABLE FOO_AVRO WITH (VALUE_FORMAT='AVRO') AS SELECT * FROM COMPUTER_T;
CTAS_FOO_AVRO_4 (FAILED): INSERT INTO FOO_AVRO VBLAH BLAH BLAH
```
